### ISSUE TITLE
Final Peer Review - rz98

### ISSUE THREAD

**Issue by ZRich098** [Meiqi-Wu/vaccine-discovery#11]
The vaccine discovery project attempted to determine whether they could predict COVID-19 antibody valence to help biologists find segments of the virus that creates antibodies in humans. The group used antibody valence data from B cell proteins and SARS virus proteins, including factors such as amino acid sequences, accessible surface areas, etc.  
The group analyzed the data to create comparisons of isoelectric point distributions and related features, which influenced the types models they decided to build.  The group used models such as logistic regression, random forest, and dense neural network to predict valid protein antibody valences.  By doing so, the group was able to calculate 2700 COVID protein sequences to investigate that could result in a possible vaccine. 

Three things I liked about the project are:
1. The project is both relevant to current events and very much humanitarian.  The group was able to take a real life problem and produce results that could contribute to solving the current COVID crisis, namely identifying potential protein segments and determining important protein features related to determining antibody valence. 
2. The very in depth explanation of each of the methods we did not cover in class was very much appreciated.  The paper walks through the different biology terms we needed to know as background knowledge, mathematical formulas used in each of the models for clarification, as well as detailed explanations of models such as random forest, gradient boosting, and dense neural networks. 
3. Clearly acknowledging the extent of the modeling done and covering potential future directions to take the research.  The paper notes that the models could have been improved by using more complex features and avoiding the black box nature of certain models, but also explains the difficulty in producing complex features and modeling may be beyond the scope of the course. 

Three areas I think could be improved:
1. The project avoided looking into bias and fairness in the models. There could have been discussion on whether the fairness of training the models using the data from B cell proteins and SARS virus proteins, or whether doing so introduced bias in some way.
2. Given that determining important features was a project goal, what is the group consensus on the result? The logistic regression model clearly valued start and end position way above all other features, while the other models did valued other features more highly.  Should the most accurate model (Random Forest) be taken as truth, even though it is more black-box?
3. The paper could have gone more in depth about how they settled on their hyperparameters in models such as Random Forest, Gradient Boosting, and Neural Networks .  The paper mentioned that grid search was used to determine tree height in Random Forest, but don't mention any other hyperparameters such as splitting conditions, number of samples, hidden layer size, etc.
